File: baekjoon/1929/1929.c
#include <stdbool.h>
#include <stdio.h>
#include <math.h>

bool prime(int num)
{
    if(num == 1)
        return false;
    
    int n = sqrt(num);

    for(int i = 2; i < n + 1; i++)
        if(num % i == 0)
            return false;
    return true;
}

int main()
{
    int m, n;

    scanf("%d %d", &m, &n);

    for(int i = m; i < n + 1; i++)
        if(prime(i))
            printf("%d\n", i);
}

File: baekjoon/1929/1929.py
import math

def prime(num):
    if num == 1:
        return False
    
    n = int(math.sqrt(num))

    for i in range(2, n + 1):
        if num % i == 0:
            return False
    return True

m, n = map(int, input().split())

for k in range(m, n + 1):
    if prime(k) : print(k)